### ISSUE TITLE
[Feature request] Add an option to see connected session and the ability to disconnect all sessions

### ISSUE THREAD

**Issue by samy570** [FilenCloudDienste/filen-drive#82]
It can be very handy to see a session has been left open on an old device or a temporary device...

**Comment by DanielProg39** [comment]
Yes, a feature to view and manage active sessions would be nice. I've seen this in Mega and it's very useful, giving confidence that no one is secretly accessing files. One thing though, there should be some measures taken to prevent abuse, such as the ability to end a session or all sessions should only be available after some time after logging in, and a password check should be made before allowing this.
